RevenueCatUI/Views/FooterView.swift

```diff
@@ -114,6 +115,7 @@ struct FooterView: View {
         } label: {
             Text("All subscriptions", bundle: .module)
         }
+        .frame(minHeight: FooterView.buttonMinHeight)
```

Comment:
Should we apply this to the entire view instead?

Reply:
That was my original thought, but I want to make sure the button tap area itself is 44pt. The buttons won't always expand to fill the HStack right?

Reply:
Good point. You'd need `.frame(maxHeight: .infinity)` but that will cause other issues so this is fine 👍🏻